Dynamic RBAC.adoc

```diff
@@ -0,0 +1,320 @@
+                                "NONE"
+                            ]
+                        }
+                    }
+                },
+                "responses": {
+                    "204": {
+                        "description": "The role was created"
+                    },
+                    "409": {
+                        "description": "Role already exists"
+                    }
+                }
+            },
+            "put": {
+                "description": "Replaces an existing role-to-permission mapping",
+                "parameters": {
+                    "roleName": {
+                        "name": "roleName",
+                        "in": "path",
+                        "required": true,
+                        "schema": {
+                            "type": "string"
+                        }
+                    },
+                    "permissions": {
+                        "name": "permission",
+                        "in": "query",
+                        "required": true,
+                        "schema": {
+                            "type": "array",
+                            "enum": [
+                                "LIFECYCLE",
+                                "READ",
+                                "WRITE",
+                                "EXEC",
+                                "LISTEN",
+                                "BULK_READ",
+                                "BULK_WRITE",
+                                "ADMIN",
+                                "CREATE",
+                                "MONITOR",
+                                "ALL",
+                                "ALL_READ",
+                                "ALL_WRITE",
+                                "NONE"
+                            ]
+                        }
+                    }
+                },
+                "responses": {
+                    "204": {
+                        "description": "The role was replaced"
+                    },
+                    "404": {
+                        "description": "Role not found"
+                    }
+                }
+            },
+            "delete": {
+                "description": "Remves a role-to-permission mapping",
+                "responses": {
+                    "204": {
+                        "description": "The role was deleted",
+                    },
+                    "404": {
+                        "description": "Role not found"
+                    }
+                }
+            }
+        }
+    }
+}
+----
+
+
+== CLI
+
+* `*roles create* --permissions='perm1'[,'perm2'...] _role_` Create a new role
+* `*roles alter* --permissions='perm1'[,'perm2'...] _role_` Alters an existing role
+* `*roles remove* _role_` Remove an existing roleReplace
+* `*roles ls*` List all roles
+
+== Console
+
+The console should provide a "Global roles" view, with the ability to list, add, edit and remove roles.
+
+== Operator
```

Comment:
I would add a way we want to target as well, in the template, to disseminate this feature. Simple tutorial? Blog post? so we add a JIRA afterwards too.